### PR TITLE
Allow passing klass to specify field class to field_for

### PR DESCRIPTION
**Pull request by cancan101**
Closes https://github.com/marshmallow-code/marshmallow-sqlalchemy/issues/81

### CODE REVIEW

File: marshmallow_sqlalchemy/convert.py [hunk 1]
```diff
@@ -120,8 +120,8 @@ def fields_for_table(self, table, include_fk=False, fields=None, exclude=None, b
                 result[column.key] = field
         return result
 
-    def property2field(self, prop, instance=True, **kwargs):
-        field_class = self._get_field_class_for_property(prop)
+    def property2field(self, prop, instance=True, klass=None, **kwargs):
```
Comment (cancan101) [inline marshmallow_sqlalchemy/convert.py]: could call this `field_class` instead?

Comment (sloria) [reply marshmallow_sqlalchemy/convert.py]: +1 to `field_class`

**Review by sloria — CHANGES_REQUESTED** [review]
Looks good. Would you mind adding a test to go with this change? 

Once that's done, this is good to merge.

**Comment by cancan101** [comment]
another test? I already wrote one.

**Comment by sloria** [comment]
@cancan101 Ah, sure enough. Sorry, missed that added assertion. Great. I will get this merged and cut a release when I'm off work.

**Review by sloria — APPROVED** [review]

**Comment by cancan101** [comment]
@sloria I still need to make the field_class change

**Comment by cancan101** [comment]
ok fixed

**Pull request merged**